consensus/src/round_manager.rs

```diff
@@ -837,6 +842,25 @@ impl RoundManager {
         self.round_state.record_vote(vote.clone());
         let vote_msg = VoteMsg::new(vote.clone(), self.block_store.sync_info());
 
+        // generate and multicast randomness share for the fast path
+        if let Some(fast_config) = &self.fast_rand_config {
+            let ledger_info = vote.ledger_info();
+            if !ledger_info.is_dummy() {
```

Comment:
when is this dummy?

Reply:
When there is no 2-chain (due to leader timeouts etc). So a quorum vote on this dummy ledger info will not order any block.